packages/xo-server/src/api/host.mjs

```diff
@@ -119,15 +119,23 @@ set.resolve = {
 
 // FIXME: set force to false per default when correctly implemented in
 // UI.
-export async function restart({ bypassBackupCheck = false, host, force = true, suspendResidentVms }) {
+export async function restart({
+  bypassBackupCheck = false,
+  host,
+  force = true,
```

Comment:
Unrelated to this PR but why is `force` enabled by default? :grimacing: 

I think we should change that in another PR.

Reply:
Done https://github.com/vatesfr/xen-orchestra/commit/ce617e073283676e40f5ccc7157fb2b3193857fc